### PR TITLE
More filter expressions

### PR DESCRIPTION
**Pull request by romdeau23**
Added a few new filter expressions:

- `myilvl` - the character's equipped item level
- `rolefit ` - true if there's a spot for the character's current role, false otherwise
- `partyfit` - true if there are spots for all of the party member roles, false otherwise

These can be combined to create "smart" filters, mostly useful for m+ groups, such as:

```
partyfit and (ilvl == 0 or myilvl >= ilvl) and not declined
```

I couldn't provide non-english localizations as I don't speak any of those languages :)

### CODE REVIEW

File: Localization/deDE.lua [hunk 1]
```diff
@@ -58,12 +58,15 @@ L["dialog.tooltip.op.string"] = "Operatoren auf Zeichenketten"
 L["dialog.tooltip.op.func"] = "Funktionen"
 L["dialog.tooltip.example"] = "Beispiel"
 L["dialog.tooltip.ilvl"] = "benötigte Gegenstandsstufe"
+L["dialog.tooltip.myilvl"] = "my item level"
```
Comment (0xbs) [inline Localization/deDE.lua]:
```suggestion
L["dialog.tooltip.myilvl"] = "meine Gegenstandsstufe"
```

File: Localization/deDE.lua [hunk 2]
```diff
@@ -58,12 +58,15 @@ L["dialog.tooltip.op.string"] = "Operatoren auf Zeichenketten"
...
 L["dialog.tooltip.hlvl"] = "benötigte Ehrenstufe"
 L["dialog.tooltip.defeated"] = "Anzahl an besiegten Raidbossen"
 L["dialog.tooltip.members"] = "Anzahl an Mitgliedern"
 L["dialog.tooltip.tanks"] = "Anzahl an Tanks"
 L["dialog.tooltip.heals"] = "Anzahl an Heilern"
 L["dialog.tooltip.dps"] = "Anzahl an DDs"
+L["dialog.tooltip.rolefit"] = "has spot for my role"
```
Comment (0xbs) [inline Localization/deDE.lua]:
```suggestion
L["dialog.tooltip.rolefit"] = "hat Platz für meine Rolle"
```

File: Localization/deDE.lua [hunk 3]
```diff
@@ -58,12 +58,15 @@ L["dialog.tooltip.op.string"] = "Operatoren auf Zeichenketten"
...
+L["dialog.tooltip.partyfit"] = "has spots for my party roles"
```
Comment (0xbs) [inline Localization/deDE.lua]:
```suggestion
L["dialog.tooltip.partyfit"] = "hat Platz für die Rollen meiner Gruppe"
```

File: Localization/frFR.lua [hunk 4]
```diff
@@ -58,12 +58,15 @@ L["dialog.tooltip.op.string"] = "Operateurs Textes"
 L["dialog.tooltip.op.func"] = "Fonctions"
 L["dialog.tooltip.example"] = "Example"
 L["dialog.tooltip.ilvl"] = "niveau d'objet requis"
+L["dialog.tooltip.myilvl"] = "my item level"
```
Comment (0xbs) [inline Localization/frFR.lua]:
```suggestion
L["dialog.tooltip.myilvl"] = "mon niveau d'objet"
```

File: Localization/frFR.lua [hunk 5]
```diff
@@ -58,12 +58,15 @@ L["dialog.tooltip.op.string"] = "Operateurs Textes"
...
 L["dialog.tooltip.hlvl"] = "niveau d'honneur requis"
 L["dialog.tooltip.defeated"] = "nombre de boss de raids vaincu"
 L["dialog.tooltip.members"] = "nombre de membres"
 L["dialog.tooltip.tanks"] = "nombres de tanks"
 L["dialog.tooltip.heals"] = "nombres de soigneurs"
 L["dialog.tooltip.dps"] = "nombre de DPS"
+L["dialog.tooltip.rolefit"] = "has spot for my role"
```
Comment (0xbs) [inline Localization/frFR.lua]:
```suggestion
L["dialog.tooltip.rolefit"] = "a une place pour mon rôle"
```

File: Localization/frFR.lua [hunk 6]
```diff
@@ -58,12 +58,15 @@ L["dialog.tooltip.op.string"] = "Operateurs Textes"
...
+L["dialog.tooltip.partyfit"] = "has spots for my party roles"
```
Comment (0xbs) [inline Localization/frFR.lua]:
```suggestion
L["dialog.tooltip.partyfit"] = "a place pour les rôles de mon groupe"
```

**Comment by romdeau23** [comment]
Yeah I suppose `partyfit` should be enough

**Comment by 0xbs** [comment]
Great, awaiting the changes and will merge afterwards.

**Pull request merged**